multibranch/pom.xml

```diff
@@ -88,7 +88,7 @@ THE SOFTWARE.
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>cloudbees-folder</artifactId>
-            <version>5.2.1</version>
+            <version>5.8-SNAPSHOT</version>
```

Comment:
I think it's better with `5.8-SNAPSHOT` (if it does not introduce any issue, of course). In this way we are saving a TODO here (we already have a good amount to take care once plugins go to 2.0 baseline).